doc/PACKAGERS.md

```diff
@@ -0,0 +1,83 @@
+Packaging
+=========
+
+Thank you for taking the time to package Rizin!
+
+There are some things to consider when packaging Rizin for your distribution.
+We will look at each step below.
+
+Cloning Source
+--------------
+
+Rizin uses git submodules to track versions of dependencies. You can check out
+the Rizin source and all dependencies using
+`git clone --recursive https://github.com/rizinorg/rizin`.
+
+If you change branches run `git submodule update --init --recursive` to ensure
+you are tracking the correct versions of the submodules.
+
+Building
+--------
+
+Rizin uses the `meson` build system. Command line flags passed to meson can
+change how Rizin is built.
+
+To define the base install location for Rizin use the `--prefix` flag when
+invoking `meson`. For system installs it is common to use `/usr`. If in doubt
+check your distributions packaging guidelines.
+
+Rizin uses the Capstone disassembly engine and supports versions 3, 4, and 5.
+By default we pull version 4 and statically link it into the Rizin executables.
+Some distributions might prefer that a system version of Capstone be
+dynamically linked at runtime. To do this, use the `-Duse_sys_capstone=true`
+command line option when running `meson`. See `capstone.md` for more details.
+
```

Comment:
I would add some paragraphs to mention the other compile-time options like `use_sys_magic`, `use_sys_zip`, etc. and add a reference to https://github.com/rizinorg/rizin/blob/dev/meson_options.txt . I think they are quite important for distros that want to avoid embedded code.